tools/ci_build/build.py

```diff
@@ -151,14 +158,14 @@ def parse_arguments():
     parser.add_argument(
         "--enable_training_pipeline_e2e_tests", action="store_true",
         help="Enable the pipeline c++ e2e tests.")
-    parser.add_argument(
-        "--use_horovod", action='store_true', help="Enable Horovod.")
     parser.add_argument(
         "--disable_nccl", action='store_true', help="Disable Nccl.")
     parser.add_argument(
         "--mpi_home", help="Path to MPI installation dir")
     parser.add_argument(
         "--nccl_home", help="Path to NCCL installation dir")
+    parser.add_argument(
```

Comment:
Maybe we should consolidate --mpi_home and --use_mpi in a single flag? Or at least make explicit the allowed combinations and print warnings when possibly unintended behavior is detected

Reply:
added a warning to handle the above case